Automattic/wp-calypso
# Remove code related to the Videopress TV and Videopress TV purchase flows

Related to https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

* We are removing the "Videopress" flow (see [project thread](paYJgx-5ol-p2)), and while working on that I noticed the Videopress TV flow, which never got launched, so I'm removing it as well.

## Testing Instructions



* Open the live preview
* Go to `/setup/videopress-tv`
* Check that you are redirected to wp-admin
* Repeat the previous steps with `/setup/videopress-tv-purchase`

## Pre-merge Checklist
